addons/website_sale/models/sale_order.py

```diff
@@ -455,7 +455,7 @@ def _prepare_order_line_values(
             values['product_custom_attribute_value_ids'] = [
                 fields.Command.create({
                     'custom_product_template_attribute_value_id': custom_value['custom_product_template_attribute_value_id'],
-                    'custom_value': custom_value['custom_value'],
+                    'custom_value': custom_value.get('custom_value', ''),
```

Comment:
We receive custom value as '' when we do the same with Customizable Desk as main prodcut.

Ithink you should check here https://github.com/odoo/odoo/blob/1bb5d7cf6b110f576d2e6359ae56896ca23aa19b/addons/website_sale/static/src/js/website_sale_configurators.js#L209 Why we received `ptal.customValue` as undefined